tools/llvm-spirv/llvm-spirv.cpp

```diff
@@ -135,6 +135,16 @@ static cl::opt<bool> SPIRVGenKernelArgNameMD(
     cl::desc("Enable generating OpenCL kernel argument name "
              "metadata"));
 
+static cl::opt<SPIRV::ExtInst>
+    ExtInst("spirv-ext-inst",
+            cl::desc("Specify external instructions to use when "
+                     "translating from LLVM IR to SPIR-V"),
+            cl::values(clEnumValN(SPIRV::ExtInst::None, "none",
```

Comment:
Please specify, that otherwise the LLVM intrinsic will be emulated

Reply:
Done